packages/cli/package.json

```diff
@@ -21,7 +21,7 @@
     "lint:commit": "eslint --cache $(git diff --relative --name-only --diff-filter=ACMRTUXB master | grep  -E \"(.js$|.ts$|.tsx$)\")",
     "lint:fix": "eslint \"src/**/*.{js,ts,tsx}\"  --cache --fix",
     "cli": "node dist/main.js",
-    "cli:debug": "echo '\n\nVisit chrome://inspect in Chrome and attach the debugger\n\n' && node --inspect-brk dist/main.js",
+    "cli:debug": "echo '\n\nVisit chrome://inspect in Chrome and attach the debugger\n\n' && METICULOUS_NO_TIMEOUT=true METICULOUS_REPLAY_NODE_DEBUGGER_ENABLED=true node --inspect-brk dist/main.js",
```

Comment:
`METICULOUS_REPLAY_NODE_DEBUGGER_ENABLED` breaks within replay code and assigns useful state to the window, so you're ready to debug replay code.